app/locale/fr.coffee

```diff
@@ -3160,45 +3160,45 @@ module.exports = nativeDescription: "français", englishDescription: "French", t
     teacher_email: "Adresse électronique de l'enseignant(e)"
 
 #  school_administrator:
```

Comment:
this needs to be uncommented too. A similar change for interactives, admin etc below. The build is failing because of that I think.